sormas-app/app/src/main/java/de/symeda/sormas/app/util/DiseaseConfigurationCache.java

```diff
@@ -43,6 +43,10 @@
 	private List<Disease> caseBasedDiseases = new ArrayList<>();
 	private List<Disease> aggregateDiseases = new ArrayList<>();
 	private List<Disease> followUpEnabledDiseases = new ArrayList<>();
+
+	private Map<Disease, Boolean> extendedClassificationDiseases = new HashMap<>();
```

Comment:
I don't think we need maps here, lists that contains all diseases using the extended classifications should be enough and can then be used with `contains`

Reply:
changed to list with contains

Reply:
actually we stick to maps, thank you @MateStrysewske :)